modules/digitaluniverse/rendering/renderablebillboardscloud.cpp

```diff
@@ -204,15 +207,31 @@ namespace {
         "object."
     };
 
+    static const openspace::properties::Property::PropertyInfo CorrectionSizeEndDistanceInfo = {
+        "CorrectionSizeEndDistance",
+        "Distance in 10^X meters where correction size stops acting.",
+        "Distance in 10^X meters where correction size stops acting."
+    };
+
+    static const openspace::properties::Property::PropertyInfo CorrectionSizeFactorInfo = {
+        "CorrectionSizeFactor",
+        "Control variable for distance size.",
+        ""
+    };
+    static const openspace::properties::Property::PropertyInfo PixelSizeControlInfo = {
+        "EnablePixelSizeControl",
+        "Enable pixel size control.",
+        "Enable pixel size control for rectangular projections."
+    };
 }  // namespace
 
 namespace openspace {
 
-documentation::Documentation RenderableBillboardsCloud::Documentation() {
-    using namespace documentation;
-    return {
-        "RenderableBillboardsCloud",
-        "digitaluniverse_RenderableBillboardsCloud",
+    documentation::Documentation RenderableBillboardsCloud::Documentation() {
```

Comment:
Indentation in the entire file

Reply:
I didn't get it. What is the problem here? How should it be?